src/components/atoms/About.jsx

```diff
@@ -0,0 +1,168 @@
+import styled from 'styled-components';
+import image1 from '../../assets/image 12.png';
+import image2 from '../../assets/image 2.png';
+import image3 from '../../assets/image 3.png';
+import Body1 from '../shared/Typography/Body1';
+
+const AboutUsSection = styled.section`
+  display: flex;
+  flex-direction: column;
+  padding: 50px;
+  background-color: #f8f8f8;
+  text-align: center;
+  overflow: hidden;
+  width: 100%;
+`;
+
+const AboutUsRow = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 128px;
+  flex-wrap: wrap;
+`;
+
+const AboutUsImage = styled.img`
+  height: 300px;
+  width: 300px;
+  border-radius: 8px;
+  margin-right: 20px;
+  order: ${({ reverse }) => (reverse ? 2 : 1)};
+`;
+
+const AboutUsImage1 = styled.img`
+  width: 400px;
+  height: 400px;
+  border-radius: 8px;
+  margin-right: 20px;
+  order: ${({ reverse }) => (reverse ? 2 : 1)};
+`;
+
+const AboutUsText = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+
+  text-align: left;
+  order: ${({ reverse }) => (reverse ? 1 : 2)};
+`;
+const AboutUsBox1 = styled.div`
+  text-align: left;
+  order: ${({ reverse }) => (reverse ? 1 : 2)};
+  width: 60%;
+  margin-right: 48px;
+`;
+
+const AboutUsBox3 = styled.div`
+  text-align: left;
+  order: ${({ reverse }) => (reverse ? 1 : 2)};
+  width: 60%;
+  margin-right: 144px;
+  display: flex;
+  flex-direction: column;
+`;
+
+const Headi1 = styled.h1`
+  color: #2a2a2a;
+  font-family: Roboto;
+  font-size: 45px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  width: 606px;
+  height: 196px;
+  @media (max-width: 576px) {
+    font-size: 24px;
+  }
+  @media (max-width: 768px) {
+    font-size: 36px;
+  }
+`;
+
+const Headi2 = styled.h1`
+  width: 894px;
+  height: 186px;
+  color: #2a2a2a;
+  font-family: Roboto;
+  font-size: 45px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  @media (max-width: 576px) {
+    font-size: 24px;
+  }
+  @media (max-width: 768px) {
+    font-size: 36px;
+  }
+`;
+
+const Headi3 = styled.h1`
+  width: 613px;
+  color: #2a2a2a;
+  font-family: Roboto;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+
+  @media (max-width: 576px) {
+    font-size: 24px;
+  }
+`;
+
+const AboutUs = () => {
+  return (
+    <AboutUsSection>
+      <AboutUsRow>
+        <AboutUsBox1>
+          <AboutUsText reverse>
```

Comment:
Why is all these needed? Use simple n-th child selector in CSS. Refer [this](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)